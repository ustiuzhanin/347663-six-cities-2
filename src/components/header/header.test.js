import React from 'react';
import Header from './header.jsx';
import ShallowRenderer from 'react-test-renderer/shallow';

test(`Login's Snapshot`, () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Header />);
  const result = renderer.getRenderOutput();

  expect(result).toMatchSnapshot();
});
