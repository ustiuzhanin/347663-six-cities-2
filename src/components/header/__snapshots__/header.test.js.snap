// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header's Snapshot 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      >
        <ForwardRef
          className="header__logo-link"
          to="/"
        >
          <img
            alt="6 cities logo"
            className="header__logo"
            height="41"
            src="/img/logo.svg"
            width="81"
          />
        </ForwardRef>
      </div>
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        >
          <React.Fragment>
            <ProfileBtn
              title="Login"
              url="/login"
            />
            <ProfileBtn
              title="Sign Up"
              url="/signup"
            />
          </React.Fragment>
        </ul>
      </nav>
    </div>
  </div>
</header>
`;

exports[`Login's Snapshot 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      >
        <ForwardRef
          className="header__logo-link"
          to="/"
        >
          <img
            alt="6 cities logo"
            className="header__logo"
            height="41"
            src="/img/logo.svg"
            width="81"
          />
        </ForwardRef>
      </div>
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        >
          <React.Fragment>
            <ProfileBtn
              title="Login"
              url="/login"
            />
            <ProfileBtn
              title="Sign Up"
              url="/signup"
            />
          </React.Fragment>
        </ul>
      </nav>
    </div>
  </div>
</header>
`;
